blosc2/blosc2_ext.pyx

```diff
@@ -1819,10 +1869,32 @@ def meta_keys(self):
     return keys
 
 
-def open(urlpath, mode, offset, **kwargs):
+def open(urlpath, mode, mmap_mode, initial_mapping_size, offset, **kwargs):
```

Comment:
But that important, but as mmap is a detail of the backend, I'd pack them in `kwargs`.  Agreed?

Reply:
Yes agreed, putting the new arguments into kwargs makes much more sense.